module/zfs/zfs_replay.c

```diff
@@ -860,6 +863,89 @@ zfs_replay_setattr(void *arg1, void *arg2, boolean_t byteswap)
 	return (error);
 }
 
+static int
+zfs_replay_setsaxattr(void *arg1, void *arg2, boolean_t byteswap)
+{
+	zfsvfs_t *zfsvfs = arg1;
+	lr_setsaxattr_t *lr = arg2;
+	znode_t *zp;
+	nvlist_t *nvl;
+	size_t sa_size;
+	const char *name;
+	const void *value;
+	size_t size;
+	int error = 0;
+
+	ASSERT(spa_feature_is_enabled(zfsvfs->z_os->os_spa,
```

Comment:
I think this should be `spa_feature_is_active`

Reply:
yes. Changed.